_source/logzio_collections/_log-sources/fluentd-windows.md

```diff
@@ -0,0 +1,178 @@
+data-for-product-source: Logs
+shipping-tags:
+  - log-shipper
+contributors:
+  - nshishkin
+shipping-tags:
+  - agents
+order: 180
+---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Overview](#overview)
+* [Setup instructions](#setup-instructions)
+* [Multiline logs](#multiline)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="overview">
+
+Fluentd is a data collector, which unifies the data collection and consumption. This integration allows you to use Fluentd to send logs from your Windows system to your Logz.io account. 
+
+<!-- info-box-start:info -->
+Fluentd will fetch all existing logs, as it is not able to ignore older logs.
+{:.info-box.note}
+<!-- info-box-end -->
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="setup-instructions">
+
+
+#### Configure Fluentd
+
+**Before you begin, you'll need**:
+Ruby and ruby-dev 2.1 or higher
+
+<div class="tasklist">
+
+	
+##### Install Fluentd td-agent
+
+Navigate to the [downloads](https://td-agent-package-browser.herokuapp.com/4/windows) page of td-agent and download the latest version of the installer. After that, run the installer and follow the wizard instructions.
+
+##### Install the Logz.io plugin
+
+```shell
+gem install fluentd fluent-plugin-logzio
+```
+
+##### Set up td-agent.conf
+
+Open C:/opt/td-agent/etc/td-agent/td-agent.conf and replace its contents with the following configuration:
+
+```xml
+<source>
+  @type windows_eventlog2
+  @id windows_eventlog2
+  channels application,system,security
+  read_existing_events false
+  tag winevt.raw
+  rate_limit 200
+  <storage>
+    @type local
+    persistent true
+    path C:\opt\td-agent\winlog.json
+  </storage>
+</source>
+
+<match **>
+  @type logzio_buffered
+  endpoint_url <<LISTENER-HOST>>:8071?token=<<LOG-SHIPPING-TOKEN>>&type=<<LOG-TYPE>>
+  output_include_time true
+  output_include_tags true
+  http_idle_timeout 10
+  <buffer>
+      @type memory
+      flush_thread_count 4
+      flush_interval 3s
+      chunk_limit_size 16m
+      queue_limit_length 4096
+  </buffer>
+</match>
+```
+
+###### Parameters
+
+| Parameter | Description |
+|---|---|
+| endpoint_url | A url composed of your Logz.io region's listener URL, account token, and log type. {% include log-shipping/listener-var.html %} {% include log-shipping/log-shipping-token.html %} |
+| output_include_time | To add a timestamp to your logs when they're processed, `true` (recommended). Otherwise, `false`. |
+| output_include_tags | To add the `fluentd` tag to logs, `true`. Otherwise, `false`. If `true`, use in combination with `output_tags_fieldname`. |
+| output_tags_fieldname | If `output_include_tags` is `true`, sets output tag's field name. The default is `fluentd_tag` |
+| http_idle_timeout | Time, in seconds, that the HTTP connection will stay open without traffic before timing out. |
+| type | Log type. If required, replace `<<LOG_TYPE>>` with the desired name for the log type, the default value is `fluentbit` |
```

Comment:
Should be log-type
Also should be under endpoint url and not in the end of the table